### PR TITLE
Extract and translate questionnaire completed block

### PR DESCRIPTION
**Pull request by dcdarrell9**
`"id": "questionnaire-completed"` was not being translated.

Added functionality for extracting / translating.
Regex not replacing correctly

### CODE REVIEW

File: app/translate_survey.py [hunk 1]
```diff
@@ -96,6 +96,10 @@ def translate_survey(survey_json, translations):
                     translate_titles_text(question, question['id'], translations)
                     translate_definitions_text(question, question['id'], translations)
 
+                    if 'answers' not in question:
+                        translate_container(question, question['id'], translations)
```
Comment (jonnyshaw89) [inline app/translate_survey.py]: Do we need this? the arguments are identicle to https://github.com/ONSdigital/eq-translations/pull/22/files#diff-a8244b82dd0f2093e88fb9263c12af67R93

File: app/extract_translation_data.py [hunk 2]
```diff
@@ -65,6 +65,10 @@ def get_text(data):
                     translatable_text.extend(get_question_translatable_text(question))
                     translatable_text.extend(get_definitions_text(question))
 
+                    if 'answers' not in question:
```
Comment (jonnyshaw89) [inline app/extract_translation_data.py]: Why not make this `if container['type'] == 'Content':`

File: app/translate_survey.py [hunk 3]
```diff
@@ -35,7 +35,11 @@ def dumb_to_smart_quotes(string):
     string = re.sub(r'=‘(.*?)’', r"='\1'", string)
     # Reverse: Find any SMART quotes that have been (mistakenly) placed around Jinja
     # attributes (following [) and replace them with dumb quotes.
-    string = re.sub(r'\[‘(.*?)’', r"='\1'", string)
+    string = re.sub(r'\[‘(.*?)’', r"['\1'", string)
+    # Reverse: Find any SMART quotes that have been (mistakenly) placed around date
```
Comment (bitdivision) [inline app/translate_survey.py]: This comment should be removed / updated

File: app/merge-translations.py [hunk 4]
```diff
@@ -45,13 +45,13 @@ def output_translations(translations, aFile):
     alpha_keys = set(alpha_translations.keys())
     beta_keys = set(beta_translations.keys())
 
-    for aKey in alpha_keys.difference(alpha_keys):
```
Comment (jonnyshaw89) [inline app/merge-translations.py]: What is this for?

Comment (bitdivision) [reply app/merge-translations.py]: Presumably it prints out any keys that are different between the two. Looks as though it was previously comparing one to itself and was never printing. I don't know why we need to print them though

**Pull request merged**